### PR TITLE
Add some more to `shell.mdx`

### PR DESCRIPTION
**Pull request by VehementHam**
I added descriptions for the `ln` and the `ls` command. The `ls` command also deserves a flag table, cause it's got so many useful flags.

### CODE REVIEW

File: src/content/docs/en/linux/shell.mdx [hunk 1]
```diff
@@ -41,7 +41,38 @@ To change directories, type `cd` followed by the path you want to change to.
...
+total 4
+drwxr-xr-x    2 bob bob      4096 Jan 1 13:00 dir
+-rw-r--r--    1 bob bob         0 Jan 1 12:00 file
+-rwxrwxrwx    1 ann ann      1337 Jan 1 11:00 file2
+```
+
+Symlinks are, as the name would imply, links between files. Where a symlink occurs, changes are synced between files. To create a symlink bewtween two files, type `ln`, followed by the `-s` flag, and two arguments: one being the first file, and the other being the new linked file.
+
+```
+ln -s file newfile
```
Comment (madonuko) [inline src/content/docs/en/linux/shell.mdx:73]:
```suggestion
$ ln -s file newfile
        ─┬── ───┬───
         │      └─ Path to new symlink
         └─ original file
```

File: src/content/docs/en/linux/shell.mdx [hunk 2]
```diff
@@ -41,7 +41,38 @@ To change directories, type `cd` followed by the path you want to change to.
...
 ```
 
-To list the contents of a directory, type `ls`.
+To list the contents of a directory, type `ls` followed by the path of the directory of which you want to list the contents of.
+
+```zsh
+ls /home/user/documents
+```
+
+The directory path is an optional agrument. Running `ls` on its will print the contents of the current working directory (the one you  `cd`ed into).
```
[the remark this answers is not in the document]
Comment (VehementHam) [reply src/content/docs/en/linux/shell.mdx:50]: Thanks

File: src/content/docs/en/linux/shell.mdx [hunk 3]
```diff
@@ -41,7 +41,38 @@ To change directories, type `cd` followed by the path you want to change to.
...
+```
+total 4
+drwxr-xr-x    2 bob bob      4096 Jan 1 13:00 dir
+-rw-r--r--    1 bob bob         0 Jan 1 12:00 file
+-rwxrwxrwx    1 ann ann      1337 Jan 1 11:00 file2
+```
+
+Symlinks are, as the name would imply, links between files. Where a symlink occurs, changes are synced between files. To create a symlink bewtween two files, type `ln`, followed by the `-s` flag, and two arguments: one being the first file, and the other being the new linked file.
+
+```
```
[the remark this answers is not in the document]
Comment (VehementHam) [reply src/content/docs/en/linux/shell.mdx:72]: For consistency that should be
`````
```zsh
`````

[the remark this answers is not in the document]
Comment (madonuko) [reply src/content/docs/en/linux/shell.mdx:72]: alright

File: src/content/docs/en/linux/shell.mdx (hunk 1 above)
Comment (madonuko) [reply src/content/docs/en/linux/shell.mdx:73]: I mean the prompt isn't important, so you can remove it

**Comment by nothingneko** [comment]
Resolve those last two and I think we're good to merge!

**Comment by VehementHam** [comment]
Actually, let me add the `cp` instructions.

**Comment by VehementHam** [comment]
Done

**Comment by VehementHam** [comment]
As long as the commits look good, I think that the purpose of this PR has been resolved.

**Comment by madonuko** [comment]
Amazing, thanks for the docs ❤️

**Pull request merged**